data/json/starting_missions.json

```diff
@@ -721,7 +721,8 @@
   {
     "id": "MISSION_LAST_DELIVERY",
     "type": "mission_definition",
-    "name": { "str": "Reach The Mansion And Finish This Delivery" },
+    "name": { "str": "Reach Your Vehicle’s Delivery Destination" },
+    "description": "According to your truck’s saved GPS coordinates, its original mission had been to deliver Food Place products to a mansion in bumfuck nowhere.  You’re not sure who orders takeout while the United States is imploding upon itself, but it seems like as good of a place as any to head to now that evacuation seems to be off the cards.  They’re rich people, after all.  Perhaps they’ll have some room for charity.",
```

Comment:
I think we can keep the mission being a stupid out of place delivery rather than a more pragmatic survival plan, that's kinda the point of Foodplace: it's carzy. Especially if you start as the True Foodperson, then you're not trying to escape you're actually trying to deliver the food for real.

Reply:
Agreed, this is the Silly Scenario. Let's leave it Silly. If you don't like the Silly Scenario as a whole then that's a whole other thing that can be handled in a different PR.